setup.cfg

```diff
@@ -62,6 +62,8 @@ disallow_untyped_defs = False
 [mypy-raiden.network.*]
 disallow_untyped_defs = False
 # ... but parts are
+[mypy-raiden.api.python]
```

Comment:
this one should probably move before the `[mypy-raiden.network.*]` line, so the comment in 64 stays true

Reply:
I think this is correct as it is. We set `disallow_untyped_defs = True` in https://github.com/raiden-network/raiden/pull/5096/files#diff-380c6a8ebbbce17d55d50ef17d3cf906R49

However, not the while `raiden.api` module is typed yet, so we disable it for that module in https://github.com/raiden-network/raiden/pull/5096/files#diff-380c6a8ebbbce17d55d50ef17d3cf906R56

Now the `raiden.api.python` module is typed completely, so to prevent regressions, we enable strict typing for this module again.

Reply:
sorry, my bad. nvm my comment!